Packs/CommonScripts/ReleaseNotes/1_6_52.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### IncidentFields
+- Fixed an error where this script would fail if incident fields were not properly configured.
+- Added README.md to IncidentFields
```

Comment:
```suggestion
```
no need, in RNs we only put changes that are visible to the customers through the marketplace UI.